_pages/qa/fmla-family-and-medical-leave-act.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: "q-and-a"
+permalink: "/fmla-family-and-medical-leave-act/"
+title: What is the Family and Medical Leave Act?
+tags: fmla, family and medical leave act, maternity leave, paternity leave, parental leave, leave
+---
+
+# {{ page.title }}
+
+The Family and Medical Leave Act allows most federal employees to take up to 12 work weeks of unpaid leave per 12-month period for the birth/care of a child, an adoption or placement of child for Foster Care, the care of a spouse/son/daughter/parent with serious health condition, a serious health condition of an employee, or a qualifying exigency from spouse/son/daughter/parent of an employee on covered active duty in the Armed Forces.
+
+The Department of Labor has a [detailed guide](http://www.dol.gov/whd/fmla/) to FMLA, including this [FAQ](http://www.dol.gov/whd/fmla/fmla-faqs.htm) which goes into extensive detail.
```

Comment:
`[detailed guide to the FMLA]` and `[FMLA FAQ]` would be more accessible, methinks.